### DISCUSSION TITLE
Android: Allow pinning a specific note to the home screen

### DISCUSSION THREAD

**Discussion by vHanda** [GitJournal/GitJournal#838] [Feature Requests]

**Comment by vHanda** [comment]
Blocked by https://github.com/flutter/flutter/issues/26134

There is a small proof of concept in https://github.com/imReker/FlutterHomeWidget but I'd prefer to wait until it becomes a proper package instead of doing that myself.

**Comment by sherbang** [comment]
Would this be easier to implement as shortcuts instead of widgets?

https://stackoverflow.com/a/69893333
https://pub.dev/packages/quick_actions

**Comment by Morpheus0x** [comment]
It appears that there is a relatively new package that adds widgets in flutter: [https://pub.dev/packages/home_widget](https://pub.dev/packages/home_widget), if that is what you are waiting for. Since iOS now also has the same ability to add widgets, this could also work cross-platform.

I am in the process of moving away from Nextcloud (separation of concerns, slow php with many plugins) and I'm searching for an alternative to the Nextcloud Notes app on android. 
The only feature I am missing in GitJournal is the ability to add a note to my home screen as a widget.

The widget should have the option to have the title at the top, or not. And the ability to change the base font size of the rendered markdown in the widget. The whole note should be shown in the widget and if it doesn't fit, it should be able to scroll vertically. Clicking on the widget should just open the corresponding note, having the text cursor at the position clicked isn't necessary imo.
